support-frontend/assets/stylesheets/gu-sass/typography.scss

```diff
@@ -8,7 +8,7 @@
 // ----- Font Families ----- //
 
 $gu-titlepiece: 'GT Guardian Titlepiece', Georgia, serif !default;
-$gu-headline: 'Guardian Headline', Georgia, serif !default;
+$gu-headline: 'Guardian Egyptian Web', Georgia, serif !default;
```

Comment:
this change is because frontend calls what we call Guardian Headline, Guardian Egyptian Web

Reply:
can we get frontend to rename their font? that's the wrong name 😭